### PR TITLE
Add docs for `VictoryLabel` and `VictoryAnimation`

### PR DESCRIPTION
**Pull request by david-davidson**
Does what it says on the tin! It also DRYs things up by moving from two lists of components in the docs site (`docs/components/component-docs.jsx`, `docs/config.json`) to one: `docs/config.js`.

This is blocked by https://github.com/FormidableLabs/victory-label/pull/23, so I'll end up bumping the `victory-label` version before merging, too.

/cc @paulathevalley @coopy @boygirl

### CODE REVIEW

File: docs/components/docs.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ import Sidebar from "./sidebar";
 class Docs extends React.Component {
 
   componentWillMount() {
-    ga.pageview("/victory/docs");
+    ga.pageview(`${routingConfig.base}/docs`);
```
Comment (david-davidson) [inline docs/components/docs.jsx]: This is just a leftover change from the `projects.formidablelabs.com/victory` => `victory.formidable.com` move.

File: package.json [hunk 2]
```diff
@@ -57,6 +57,7 @@
     "raw-loader": "^0.5.1",
     "react": "0.14.x",
     "react-addons-test-utils": "^0.14.0",
+    "react-docgen": "^2.4.0",
```
Comment (david-davidson) [inline package.json]: Required to `import` the label and animation docs.

File: docs/components/docs.jsx (hunk 1 above)
Comment (coopy) [inline docs/components/docs.jsx]: good catch!

**Comment by coopy** [comment]
:+1:

**Comment by boygirl** [comment]
Approved! Just merged FormidableLabs/victory-label#23 so you should be good to go

**Pull request merged**